src/views/worktray-view/worktray-view.tsx

```diff
@@ -11,68 +11,77 @@ import {
   Spinner,
   Text,
 } from "@mtfh/common/lib/components";
-import { useAxiosSWR } from "@mtfh/common/lib/http";
 
 import { WorktrayControls, WorktrayFilters, WorktrayList } from "../../components";
 import { WorktrayURLProvider } from "../../context/worktray-context";
-import { config } from "../../services";
 import locale from "../../services/locale";
-import { StaffResults } from "../../types/staff";
 
 import "./styles.scss";
 
 const getCookieValue = (name) =>
   document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || "";
 
 export const WorktrayView = (): JSX.Element => {
+  const [showSpinner, setShowSpinner] = React.useState(true);
+  const [errorMessage, setErrorMessage] = React.useState("");
+  const [assignedPatch, setAssignedPatch] = React.useState<Patch>();
   const token = getCookieValue("hackneyToken");
   const { email: emailAddress } = token
     ? (jwt_decode(token) as { email: string })
     : { email: "" };
 
-  const { data, error } = useAxiosSWR<StaffResults>(
-    `${config.searchApiUrl}/search/staff/?${stringify({
-      searchText: emailAddress,
-    })}`,
-  );
+  useEffect(() => {
+    setShowSpinner(true);
+    getAllPatchesAndAreas()
+      .then((data) => {
+        const patch = data.filter(
+          (patchOrArea) =>
+            patchOrArea.responsibleEntities[0].contactDetails.emailAddress ===
+            emailAddress,
+        )[0];
+        setAssignedPatch(patch);
+      })
+      .catch((e) => {
+        setErrorMessage(e.message);
+      })
+      .finally(() => {
+        setShowSpinner(false);
+      });
+  }, [emailAddress]);
```

Comment:
Does this not throw an eslint error due to the missing dependencies?

Reply:
ESLint was what ordered me to include it, and it seems to work fine locally so I'm happy